calendar/interface/exchangeEvent/mivExchangeEvent.js

```diff
@@ -146,7 +146,7 @@ mivExchangeEvent.prototype = {
     },
 
     set startDate(aValue) {
-        if (aValue.toString() != this.startDate.toString()) {
+        if (aValue == null || this.startDate == null || aValue.toString() != this.startDate.toString()) {
```

Comment:
I am not sure, if we should allow setting the value of null or prevent it. Currently however, this code leads to an exception, when setting null as startDate, which aborts the whole synchonization.